test/duct/core_test.clj

```diff
@@ -74,6 +74,12 @@
 (defmethod ig/init-key ::bar [_ {:keys [x]}]
   #(update % ::x (fnil conj []) x))
 
+(defmethod ig/init-key :duct.module/routes [_ {:route/keys [resource]}]
+  #(assoc-in % [:duct/router :routes] resource))
+
+(defmethod ig/init-key :value/inc [_ v]
```

Comment:
Can you change the key name to `::inc` to make it clear that this is a key associated with the test?